### ISSUE TITLE
Spring Boot 3.2.0 regressions.

### ISSUE THREAD

**Issue by mshima** [jhipster/generator-jhipster#24533]
##### **Reproduce the error**

- [x] [ms-mf-ng-consul-oauth2-neo4j](https://github.com/jhipster/generator-jhipster/actions/runs/7167135252/job/19514804394): `Dump docker logs`/`blog`
```
  SLF4J: Class path contains multiple SLF4J providers.
  SLF4J: Found provider [org.slf4j.jul.JULServiceProvider@38cccef]
  SLF4J: Found provider [ch.qos.logback.classic.spi.LogbackServiceProvider@5679c6c6]
  SLF4J: See https://www.slf4j.org/codes.html#multiple_bindings for an explanation.
  SLF4J: Actual provider is of type [org.slf4j.jul.JULServiceProvider@38cccef]
  Error: Exception in thread "main" java.lang.IllegalArgumentException: LoggerFactory is not a Logback LoggerContext but Logback is on the classpath. Either remove Logback or the competing implementation (class org.slf4j.jul.JDK14LoggerFactory loaded from file:/app/libs/slf4j-jdk14-2.0.9.jar). If you are using WebLogic you will need to add 'org.slf4j' to prefer-application-packages in WEB-INF/weblogic.xml: org.slf4j.jul.JDK14LoggerFactory
  	at org.springframework.util.Assert.instanceCheckFailed(Assert.java:592)
  	at org.springframework.util.Assert.isInstanceOf(Assert.java:511)
  	at org.springframework.boot.logging.logback.LogbackLoggingSystem.getLoggerContext(LogbackLoggingSystem.java:396)
  	at org.springframework.boot.logging.logback.LogbackLoggingSystem.beforeInitialize(LogbackLoggingSystem.java:124)
  	at org.springframework.boot.context.logging.LoggingApplicationListener.onApplicationStartingEvent(LoggingApplicationListener.java:238)
  	at org.springframework.boot.context.logging.LoggingApplicationListener.onApplicationEvent(LoggingApplicationListener.java:220)
  	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:178)
```

- [ ] Daily builds - [Microservices JWT](https://github.com/hipster-labs/jhipster-daily-builds/actions/runs/7163116297/job/19501110129#step:20:51888)
```
2023-12-11T05:08:23.890Z  WARN 6132 --- [oundedElastic-3] reactor.core.Exceptions                  : throwIfFatal detected a jvm fatal exception, which is thrown and logged below: 
[0] 
[0] java.lang.NoClassDefFoundError: com/fasterxml/jackson/databind/util/EnumValues
[0] 	at com.fasterxml.jackson.databind.ser.std.EnumSerializer.construct(EnumSerializer.java:101)

```

##### **Browsers and Operating System**

- [ ] Checking this box is mandatory (this is just to show you read everything)

**Comment by atomfrede** [comment]
I tried to reproduce, but having heavy issues with pulling the keycloak image, either the tls handshake to quay fails directly or pulling some layers eventually times out. Has anyone experienced similar issues before?
Anyway, I think I have found the the problem with multiple slf4j binding, but not sure why it just manifest in the concrete example, will further investigate later today.

**Comment by mshima** [comment]
Fixed with 3.2.1 support.
